query/src/main/scala/filodb/query/exec/PeriodicSamplesMapper.scala

```diff
@@ -38,11 +38,12 @@ final case class PeriodicSamplesMapper(start: Long,
   protected[exec] def args: String =
     s"start=$start, step=$step, end=$end, window=$window, functionId=$functionId, funcParams=$funcParams"
 
-
+  //scalastyle:off
```

Comment:
Again let's not just use off without a specific reason to exclude it.  Try not to even turn off scalastyle, the reason it exists is to enforce good style, which this PR really needs

Reply:
I turned it off because the method is longer than 50 lines and scalastyle fails without it

Reply:
Added a comment for this